ui/scripts/vpc.js

```diff
@@ -4184,7 +4184,7 @@
 
                         // Get VPN gateways
                         $.ajax({
-                            url: createURL('listVpnGateways'),
+                            url: createURL('listVpnGateways&listAll=true'),
                             async: false,
                             data: {
                                 'vpcid': args.context.vpc[0].id
```

Comment:
Can you put all the args defined with the url itself in the data object (like vpcid is defined for example, you can add 'listAll': true)